### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by rehanullah748**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-bar">
+        <a href="#" class="logo">My logo</a>
+        <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav> 
+        </header>
```
Comment (Abenezer-Tilahun) [inline index.html:14]: - [ ] Good job for using `header` tag for toolbar section , `a` for logo and `nav` tag for hamburger menu. But for hamburger menu you try to import `menu.png` form images folder unfortunately there is no images folder in your Repo, so please try to fix it
review
.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <div class="image-icons">
+            <ul>
+            <li><div class="left-ellipse"><img src="images/Ellipse11.png" alt=""></div></li>
+            <li><div class="center-ellipse1"><img src="images/Ellipse5.png" alt=""></div></li>
+            <li><div class="center-rectangle"><img class="center-rect" src="images/Rectangle7.png" alt=""></div></li>
+            <li><div class="center-ellipse2"><img src="images/Ellipse08.png" alt=""></div></li>
+            <li><div class="right-ellipse"><img src="images/Ellipse3.png" alt=""></div></li>
+            <li><div class="right-rectangle"><img src="images/Rectangle4.png" alt=""></div></li>
+        </ul>
+        </div>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] For background image you try to import images from images folder but unfortunately there is no images folder in your Repo, so please try to fix it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            </div>
+            <div class="social-contact">
+                <h2 class="header">LET'S CONNECT</h2>
+                <a href=""><img class="social-icons" src="images/twitter.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/medium.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/github.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/angelist.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/angellist.png" alt=""></a>
+            </div>
+        </main>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Please try to use `section` tag for container of headline section.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            <p class="paragraph">I’m a software developer! I can help you <br> build a product , feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br>  have a project you need coded, don’t <br> hestiate to contact me.</p>
+            </div>
+            <div class="social-contact">
+                <h2 class="header">LET'S CONNECT</h2>
+                <a href=""><img class="social-icons" src="images/twitter.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/medium.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/github.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/angelist.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/angellist.png" alt=""></a>
+            </div>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] for the list of Social icons please try to uses a `ul` tag to group them, and `li` per icon; each icon  must has a link `a` tag.
- [ ] for `a` tag please try to  connect your social networks with the icons, using `href`.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        </main>
+        <div class="circle-bottom"><img src="images/Ellipse15.png" alt=""></div>
+        <div class="ellipse-bottom-center"><img src="images/Ellipse4.png" alt=""></div>
+        <div class="ellipse-bottom-right"><img src="images/Ellipse12.png" alt=""></div>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] For background image you try to import images from images folder but unfortunately there is no images folder in your Repo, so please try to fix it.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    
+</body>
+</html>
+
```
Comment (Abenezer-Tilahun) [inline index.html]: - Please try to  align the elements according to the requirement, you should use Flexbox in all 2 sections(toolbar and headline section.)

- Both toolbar and headline section must have the margins (top, bottom, left, right) defined in the Template design.
-    You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your project looks like this
Backgr

The Figma design looks like :

Figma design

File: style.css [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+.center-rect {
+  width: 50px;
+}
+
+.social-contact {
+  margin-left: 20px;
+  margin-top: 10px;
+}
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] Good job for implementing styling pages but please try to stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

File: README.md [hunk 8]
```diff
@@ -1 +1,74 @@
...
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (Abenezer-Tilahun) [inline README.md]: - [ ] Good job for proffesional README.
- [ optional] README file should be descriptive and all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo and please try to customize.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @rehanullah748  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.

almost

## Required Changes ♻️
_Check the comments below the review._

- [x] README file should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in your Repo. 
- [x] Please try to use meaningful Commit messages in your feature project.(at this moment you are not required to update the commit messages)
- [ ] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @rehanullah748,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job getting approved on your first try!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**